docs/vm-ee-guide.md

```diff
@@ -0,0 +1,324 @@
+    Operation:       Update
+    Time:            2023-04-17T19:50:18Z
+  Resource Version:  556954
+  UID:               f0309a24-632b-4ebd-bbeb-9b205151a311
+Spec:
+  Endpoints:
+    Ip:           10.0.0.4
+    Ip:           20.245.116.185
+  External Node:  nephe-controller
+Events:           <none>
+```
+
+### Labels
+
+| Label                   | Description                                                                               | Azure                                                            | AWS                                              |
+|-------------------------|-------------------------------------------------------------------------------------------|------------------------------------------------------------------|--------------------------------------------------|
+| nephe.io/cloud-vm-uid   | Unique identifier of VM on cloud                                                          | VM `vmId` property on cloud                                      | Instance ID of VM on cloud                       |
+| nephe.io/cloud-vpc-uid  | Unique identifier of VPC on cloud                                                         | VNET `resourceGuid` property on cloud                            | VPC ID of VPC on cloud                           |
+| nephe.io/cloud-vpc-name | User assigned VPC name on cloud                                                           | VNET `name` property on cloud                                    | VPC `Name` tag on cloud                          |
+| nephe.io/cloud-vm-name  | User assigned VM name on cloud                                                            | VM `name` property on cloud                                      | VM `Name` tag on cloud                           |
+| nephe.io/owner-vm       | VM object Name within Kubernetes cluster                                                  | VM `name` property suffixed with ascii sum of VM Resource ID     | Instance ID of VM on cloud                       |
+| nephe.io/owner-vm-vpc   | VPC object(belongs to VM) Name within Kubernetes cluster                                  | VNET `name` property suffixed with ascii sum of VNET Resource ID | VPC ID of VPC on cloud                           |
+ | nephe.io/cloud-region   | Region configured in `CPA` and VMs are imported from this region                          | Region from configured `CloudProviderAccount` CR                 | Region from configured `CloudProviderAccount` CR |
+| nephe.io/tag-Name       | Prefix "tag-" indicates VM Tags from cloud, value of label is the corresponding tag value | Tags on cloud with key name "Name"                               | Tag on cloud with key name "Name"                |
+
+### Cloud Tags
+
+The tags on the `VirtualMachine` object are imported as labels in `ExternalEntity` CR. Label name starting with a prefix
+"tag-" are the `tags` from imported `VirtualMachine` objects. In the above example, `nephe.io/tag-Cloud` and
```

Comment:
its nephe.antrea.io prefix now.. please take new outputs

Reply:
yes, all outputs are updated to include nephe.antrea.io. Also, ces-name, ces-namespace labels are included in vm objects